okteto-pipeline.yml

```diff
@@ -1,5 +1,7 @@
 icon: https://apps.okteto.com/movies/icon.png
 deploy:
+  - okteto build -t okteto.dev/api:dev api
```

Comment:
Will this redeploy the pods on a helm upgrade?

Reply:
maybe we can add an annotation with the commit to force a redeploy each time? that helps to show the commit -> redeploy cycle.